src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/Envelope.java

```diff
@@ -96,6 +103,8 @@ public Envelope(
         this.openingDate = openingDate;
         this.zipFileCreateddate = zipFileCreateddate;
         this.zipFileName = zipFileName;
+        this.classification = (classification == null ? Classification.EXCEPTION : classification);
+        this.urgent = (urgent == null ? false : urgent);
```

Comment:
See below comment for classification field

Reply:
See above comment about accepting documents using the "old" model.